sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -1377,9 +1379,16 @@ def start_copy_from_url(self, source_url, metadata=None, incremental_copy=False,
             premium storage accounts.
         :param bool requires_sync:
             Enforces that the service will not return a response until the copy is complete.
-        :param bool polling: A poller will be used for this operation. Defaults to True.
-        :returns: A pollable object to check copy operation status (and abort).
-        :rtype: :class:`~azure.storage.blob.polling.CopyStatusPoller`
+        :returns: A dictionary of copy properties (etag, last_modified, copy_id, copy_status).
+        :rtype: Dict[str, Union[str, datetime]]
+
+        Example:
+            .. literalinclude:: ../tests/test_blob_samples_common.py
+                :start-after: [START copy_blob_from_url]
+                :end-before: [END copy_blob_from_url]
+                :language: python
+                :dedent: 12
```

Comment:
should dedent be 8? I see 8 dedent in some places.

Reply:
12 is correct. It depends on how indented the code sample is and how many dedents are needed to avoid excess whitespace.